doc/source/_static/numpy.css

```diff
@@ -0,0 +1,141 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap');
+
+.main {
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+}
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 16px;
+  line-height: 150%;
+}
+
+.container {
+  width: 100%;
+}
+
+div.top-scipy-org-logo-header {
+  background-color: #fafafa;
+  border-bottom: 2px solid #013243; /* warm black */
+  margin-top: 0;
+  box-shadow: none;
+}
+div.top-scipy-org-logo-header img {
+  height: 110px;
+  padding-left: 27px;
+}
+
+div.spc-navbar .nav-pills > li > a {
+  background-color: #4d77cf; /* han blue */
+}
+
+div#spc-section-body h1, h2, h3, h4, h5, h6 {
+  font-family: 'Lato', sans-serif;
+}
+
+dt:target, span.highlighted {
+  background-color: #fff7e6; /* diluted mustard */
+}
+
+div#spc-section-body h1 {
+  color: #013243; /* warm black */
+  font-weight: 700;
+  font-size: 320%;
+  text-align: right;
+  letter-spacing: -2px;
+  word-spacing: .03em;
+  margin-top: 4.5rem;
+  margin-bottom: 4.9rem;
+  line-height: 110%;
+}
+
+div#spc-section-body h2 {
+  color: #4d77cf; /* han blue */
+  letter-spacing: -1px;
+  font-size: 200%;
+  font-weight: 400;
+  word-spacing: .05em;
+  margin-top: 2.5rem;
+}
+
+div#spc-section-body h3 {
+  letter-spacing: -1px;
+  word-spacing: .05em;
+}
+
+div#spc-section-body li, dd {
+  line-height: 150%;
+}
+
+div#spc-section-body pre {
+  font-size: 100%;
+  line-height: 150%;
+}
+
+/* Don't print "Table of Contents" heading */
+div.spc-rightsidebar h3 {
+  display: none;
+}
+
+/* Don't show the page title in TOC */

```

Comment:
Do we want to remove this from the template, rather than just hiding it in the CDs?

Reply:
The two elements I've hidden are the unnecessary "Table of Contents" title and the unnecessary top link that's the page name; what template change would do this instead?

Reply:
I've no idea, just thought it might give you more control.

Do you know which template emits the sidebar?

Also, would these changes make more sense in the numpy Sphinx theme, wherever that lives?

Reply:
>do we want to remove this from the template?

I understand you now. I think the partitioning is OK -- all the style changes are in one place, the numpy.css file, and the only template change is to point to that file.

> would these changes make more sense in the numpy Sphinx theme, wherever that lives

My first thought was to do that, but I think we're just a veneer on the scipy theme, which we have as a submodule.

Reply:
> but I think we're just a veneer on the scipy theme, which we have as a submodule.

Yes, I was thinking of the abandoned PR below.

If scipy wants similar changes (@rgommers?), then perhaps you should edit the theme directly.

Reply:
I think there are much better Sphinx themes around (e.g. the statsmodels and pandas ones), so longer term I do think we want to get rid of the current theme and just adopt another one. The maintenance cost of a theme is quite high, so sharing the load will be helpful too.

I'm happy with the current approach, adding a `numpy.css`. With the main goal being to improve the most glaring issues (font size, and styling which matches the website better). Other improvements are nice to have, but no need to redesign half the theme I think, because at some point we'll want to get rid of it.